### PR TITLE
Create ursmahi.json

### PR DESCRIPTION
**Pull request by ursmahi**
added ursmahi.json

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4600"></a>

### CODE REVIEW

**Review by sidharthviz — APPROVED** [review]
LGTM  🎉

**Review by Hrishi5111998 — APPROVED** [review]

File: data/ursmahi.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  "bio": "Fronted Web Developer",
+
+  "tags": [
+    "Open Source",
+    "Python",
+    "ReactJs",
+    "NextJS",
+    "Javascript",
+    "Docker",
+    "HTML/CSS"
```
Comment (priyankarpal) [inline data/ursmahi.json:14]: It would be great if you add separate tags.

``` suggestion
    "HTML",
    "CSS"
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by priyankarpal — APPROVED** [review]
LGTM 👍🏻

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for your Contribution! 🎉

**Pull request merged**